.github/workflows/lint_python.yml

```
@@ -0,0 +1,28 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```
Has just been released 🙂: https://github.com/actions/checkout/releases